apps/src/code-studio/components/progress/ScriptOverview.jsx

```diff
@@ -151,6 +152,21 @@ class ScriptOverview extends React.Component {
               courseName={courseName}
               userId={userId}
             />
+            {showCalendar && viewAs === ViewType.Teacher && (
+              <div
+                className="unit-calendar-for-printing"
+                style={{
+                  height: 0,
+                  visibility: 'hidden'
+                }}
```

Comment:
I expect that putting these styles inline is going to make it hard to selectively show these. I'd recommend instead adding (or finding) a class we can use to selectively hide or show this entirely with CSS. Something like
```css
.print-only {
  height: 0;
  visibility: hidden;
}

@media print {
  .print-only {
    height: auto;
    visibility: visible;
  }
}
```

Reply:
What do you mean by make it hard? This is working.

Reply:
I feel like it's helpful to see the styling in line to make it clear this should not show instead of hiding it in another file.

Reply:
This does not seem to be working. What is wrong with doing it this way? It is currently working and this makes it more obvious that this thing is hidden.

Reply:
The problem is that doing it this way requires us to use `!important` in our CSS, which we should always avoid if at all possible

Reply:
the above example may need something like `@media screen` applied to the 'hidden' case

Reply:
Ok updated.